frame/vamm/src/tests/update_twap.rs

```diff
@@ -98,20 +121,34 @@ fn update_twap_fails_if_new_twap_is_zero() {
 		})
 }
 
+#[test]
+fn should_fail_if_twaps_are_not_reciprocal() {
+	ExtBuilder::default().build().execute_with(|| {
+		let vamm_config = default_vamm_config();
+		create_vamm(&vamm_config);
+		run_to_block(1);
+		let twap = Some(as_decimal(10_u128));
+		assert_noop!(
+			TestPallet::update_twap(0, twap, twap),
+			Error::<MockRuntime>::TwapsMustBeReciprocals
+		);
+	})
+}
+
 #[test]
 fn update_twap_fails_if_twap_timestamp_is_more_recent() {
 	let timestamp = Timestamp::MIN;
 	let timestamp_greater = Timestamp::MIN + 1;
 	let vamm_state = VammState { twap_timestamp: timestamp_greater, ..Default::default() };
 	let new_twap = Some(Decimal::from_inner(10));
+	let new_twap_reciprocal = Some(TestPallet::reciprocal_twap(&new_twap.unwrap()).unwrap());
 	ExtBuilder { vamm_count: 1, vamms: vec![(0, vamm_state)] }
 		.build()
 		.execute_with(|| {
 			// For event emission
 			run_for_seconds(timestamp);
-
 			assert_noop!(
-				TestPallet::update_twap(0, new_twap, new_twap),
+				TestPallet::update_twap(0, new_twap, new_twap_reciprocal),
```

Comment:
I think `base_twap` and `quote_twap` would be more descriptive here

Reply:
Oh, okay :+1: 
I'll change that, thanks!